### PR TITLE
Impress UI: Turn notes view and normal view buttons into big tool ite…

### PR DESCRIPTION
**Pull request by gokaysatir**
…m buttons.

Change-Id: I7088382d9319c4b8119708c55c2e7c027bc2efe1

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/images/lc_view-normal-view.svg
Comment (eszkadev) [inline browser/images/lc_view-normal-view.svg:1]: we should prefer filename based on uno command, it is the priority to get icon name and also used by other components like sidebar. id defined in nb can not match

**Review by eszkadev — CHANGES_REQUESTED** [review]

File: browser/images/lc_view-normal-view.svg
Comment (gokaysatir) [reply browser/images/lc_view-normal-view.svg:1]: But it doesn't match with the uno-command-names?

**Comment by pedropintosilva** [comment]
I have amended the commit to don't rename the svg. So it says the same name as the uno command. I have tested.

**Review by eszkadev — APPROVED** [review]
looks ok

**Comment by gokaysatir** [comment]
> this is a good improvement, thanks. As a follow up PR I would love to
> 
> * Have solely one button 'Notes View' that is a toggle (on/off). Otherwise, it's quite cumbersome for the user to understand what to press to be able to exit the notes view
> * When switching between views the Active tab shouldn't change.
> 
> Note: Currently I'm unable to exist the notes view no matter if I press the notes or the normal view buttons (on Firefox, master)

Thanks @pedropintosilva :) I will create an internal task for the additional improvements.

**Pull request merged**